app/src/main/java/de/westnordost/streetcomplete/ktx/View.kt

```diff
@@ -73,6 +74,16 @@ fun View.showTapHint(initialDelay: Long = 300, pressedDelay: Long = 600) {
     }
 }
 
+@Suppress("NOTHING_TO_INLINE")
+inline fun View.setPadding(insets: Insets) {
+    setPadding(insets.left, insets.top, insets.right, insets.bottom)
+}
+
+@Suppress("NOTHING_TO_INLINE")
+inline fun View.setMargins(insets: Insets) {
+    setMargins(insets.left, insets.top, insets.right, insets.bottom)
+}
+
```

Comment:
```suggestion
fun View.setPadding(insets: Insets) {
    setPadding(insets.left, insets.top, insets.right, insets.bottom)
}

fun View.setMargins(insets: Insets) {
    setMargins(insets.left, insets.top, insets.right, insets.bottom)
}

```

As far as I understand, inlining only really makes sense if a function has a function parameter. See also the documentation:

> If an inline function has no inlinable function parameters and no reified type parameters, the compiler will issue a warning, since inlining such functions is very unlikely to be beneficial

https://kotlinlang.org/docs/inline-functions.html#noinline

Reply:
Thanks!